modules/config-github-idp.adoc

```diff
@@ -21,7 +21,7 @@ Configuring GitHub authentication allows users to log in to {product-title} with
 
 .Procedure
 
-. From {cloud-redhat-com}, navigate to the *Clusters* page and select the cluster that you need to configure identity providers for.
+. From {console-redhat-com}, navigate to the *Clusters* page and select the cluster that you need to configure identity providers for.
```

Comment:
Previously, {cloud-redhat.com} rendered as a link to [OpenShift Cluster Manager (OCM)](https://cloud.redhat.com/openshift). Is the {console-redhat-com} attribute updated?

Reply:
It looks like the attribute is still set to `cloud-redhat-com`: https://raw.githubusercontent.com/openshift/openshift-docs/dedicated-4/modules/common-attributes.adoc

Since it's still used as an example in the [contribution guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#product-name-and-version), maybe we can leave it as `cloud-redhat-com` for now?

Reply:
They want them going to the hybrid cloud console